src/dotnet/commands/dotnet-build/BuildCommandApp.cs

```diff
@@ -85,6 +89,7 @@ private void AddCompileParameters()
             _shouldPrintIncrementalPreconditionsArgument = _app.Option(BuildProfileFlag, "Set this flag to print the incremental safety checks that prevent incremental compilation", CommandOptionType.NoValue);
             _shouldNotUseIncrementalityArgument = _app.Option(NoIncrementalFlag, "Set this flag to turn off incremental build", CommandOptionType.NoValue);
             _shouldSkipDependenciesArgument = _app.Option("--no-dependencies", "Set this flag to ignore project to project references and only build the root project", CommandOptionType.NoValue);
+            _isServiceable = _app.Option("--Serviceable", "Adds the Serviceable attribute to output binaries", CommandOptionType.NoValue);
```

Comment:
All our other options are lower cased - `--framework, `--runtime`, etc.  This should be lower cased as well.